ensembl_dbcopy/admin.py

```diff
@@ -151,6 +152,7 @@ def resubmit_jobs(self, request, queryset):
     form = SubmitForm
     add_form_template = "admin/dbcopy/submit.html"
     change_form_template = "admin/dbcopy/detail.html"
+    delete_selected_confirmation_template = "admin/dbcopy/delete_selected_confirmation.html"
```

Comment:
I thought you were about to change the directory structure to something respecting Django standards?
I understood this would imply moving / renaming templates to more canonical django overridden template patterns. e.g. - `admin/dbcopy/submit.hmtl` to be moved to `templates/ensembl_dbcopy/add_form.html`  (change inheritence accordingly)
- `admin/dbcopy/detail.html` to be moved to `templates/ensembl_dbcopy/change_form.html`
then the last one would simply have been created in the right place?

Reply:
Not really long to do in the meantime, I think. be aware there is an error in the extends for `submit.html`, it should extends `add_form.html` not `change_form.html`.

Reply:
The path changes will be in another PR then?

Reply:
No, I'm working on it. I'll commit the changes here.